src/CMakeLists.txt

```diff
@@ -30,11 +30,23 @@ mark_as_advanced(EXECUTABLE_OUTPUT_PATH LIBRARY_OUTPUT_PATH)
 option(ENABLE_CURL "Enable cURL support for fetching media" TRUE)
 set(USE_CURL FALSE)
 
+if(NOT BUILD_CLIENT)
+	set(BUILD_SERVER TRUE)
+	set(ENABLE_CURL TRUE)
```

Comment:
Cached settings?

If you do this:

    cmake . -DBUILD_SERVER=0
    cmake . -DBUILD_CLIENT=0   # -DBUILD_SERVER=0 is cached here.

Afaik.